build/distributions/CentOS/6/x86_64/yum_data/packages.yaml

```diff
@@ -278,40 +275,40 @@ simp-lastbind:
 simp-ppolicy-check-password:
   :rpm_name: simp-ppolicy-check-password-2.4.39-0.el6.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/simp-ppolicy-check-password-2.4.39-0.el6.x86_64.rpm
-simp-vendored-r10k-2.2.2-0.noarch.rpm:
+simp-vendored-r10k:
```

Comment:
rubygem-rake-compiler:  should be rubygem-rake-compiler-0.9.3-1 ?  We have 0.9.3-2 in packagecloud, but at yum.puppetlabs.com, latest is 0.9.3-1.  Need to understand the difference?